cpp/benchmarks/t/pipelines/odometry/RGBDOdometry.cpp

```diff
@@ -64,9 +64,9 @@ static void ComputeOdometryResultPointToPlane(benchmark::State& state,
     const float depth_max = 3.0;
 
     t::geometry::Image src_depth = *t::io::CreateImageFromFile(
-            std::string(TEST_DATA_DIR) + "/RGBD/depth/00000.png");
+            std::string(TEST_DATA_DIR) + "/common/RGBD/depth/00000.png");
```

Comment:
Rename `TEST_DATA_DIR` to either:
- `OPEN3D_DATA_COMMON_DIR`: equivalent to `TEST_DATA_DIR/common`
- `OPEN3D_DATA_DOWNLOAD_DIR`: equivalent to `TEST_DATA_DIR/download`

Update CMake and the corresponding code.